www/perl-lib/SMT/Rest/SCCAPIv4.pm

```diff
@@ -355,6 +355,14 @@ sub delete_single_product
 {
     my $self = shift || return (undef, undef);
 
+    my $regsharing = SMT::Utils::hasRegSharing();
```

Comment:
I've identified that registration sharing happens, we've had a meeting with @mcalmer and @rjschwei regarding the API that is used for registration sharing, we've agreed that disallowing single product deactivation when registration sharing is enabled works for now.